### PR TITLE
HTML5 and CSS3

### PR DESCRIPTION
**Pull request by SinTwo**
Port of  #https://github.com/ParadiseSS13/Paradise/pull/2497 and it's various fixes for our browser and NanoUI. Should update everything to HTML5, CSS3, and the newest version of IE as well as updating the JavaScript used in NanoUI.

### CODE REVIEW

**Comment by zekesulastin** [comment]
*glances at WINE and WinXP VM* what version of IE does this make our minimum, exactly?

File: code/modules/nano/nanoui.dm [hunk 1]
```diff
@@ -354,10 +356,11 @@ nanoui is used to open and update nano browser uis
 	var/url_parameters_json = json_encode(list("src" = "\ref[src]"))
 
 	return {"
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!DOCTYPE html>
 <html>
 	<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
 	<head>
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (SinTwo) [inline code/modules/nano/nanoui.dm]: I believe this may answer your question @zekesulastin  But otherwise I'm not entirely sure about that part.

**Comment by Datraen** [comment]
Do not merge pending investigation on running conditions for Linux users.

**Comment by zekesulastin** [comment]
Well, things /seem/ to work fine in WINE (wine_gecko, I think) and WinXP (IE 8).

**Comment by Datraen** [comment]
Omni filter UI needs to be fixed, they're near impossible to configure in their current state.

**Comment by SinTwo** [comment]
Yes, Paradise noted that it made NanoUI run much better and I noticed an improvement in using it on my testing. Also, 
communicator-html5

Kickass alpha and rounded corners.

**Comment by Datraen** [comment]
http://i.imgur.com/61rlLgk.png still needs to be fixed, however.

**Comment by SinTwo** [comment]
Fixed Omni filter and mixer. Adjusted offset for icons on crew monitoring map and camera consoles.

**Pull request merged**